src/components/Office/DefinitionLists/ShipmentInfoList.test.jsx

```diff
@@ -119,7 +123,40 @@ describe('Shipment Info List', () => {
 
     expect(screen.queryByText(labels.secondaryPickupAddress)).toBeNull();
     expect(screen.queryByText(labels.secondaryDeliveryAddress)).toBeNull();
-    expect(screen.queryByText(labels.agents[0])).toBeNull();
-    expect(screen.queryByText(labels.agents[1])).toBeNull();
+    expect(screen.queryByText(labels.mtoAgents[0])).toBeNull();
```

Comment:
```suggestion
    expect(await screen.queryByText(labels.mtoAgents[0])).not.toBeInTheDocument();
```
Can we use the above format instead for checking that something is not in the document? I find it reads a bit easier and is the preferred method for these types of checks.

**info:** [This document](https://transcom.github.io/mymove-docs/docs/frontend/testing/writing-tests-using-react-testing-library-and-jest) goes into more detail about preferred strategies when writing tests.